quarkchain/cluster/cluster_config.py

```diff
@@ -135,6 +135,8 @@ class ClusterConfig(BaseConfig):
     P2P_PORT = 38291
     JSON_RPC_PORT = 38391
     PRIVATE_JSON_RPC_PORT = 38491
+    JSON_RPC_HOST = "127.0.0.1"
```

Comment:
127.0.0.1 is the loopback ip for IPv4 and for IPv6, it should be  ::1.  localhost may be preferred as localhost will be automatically translated based on IPv4 or IPv6.